hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/client/functional/TestHoodieFileSystemViews.java

```diff
@@ -95,7 +102,14 @@ public static List<Arguments> tableTypeMetadataFSVTypeArgs() {
   @ParameterizedTest
   @MethodSource("tableTypeMetadataFSVTypeArgs")
   public void testFileSystemViewConsistency(HoodieTableType tableType, boolean enableMdt, FileSystemViewStorageType storageType, int writeVersion) throws IOException {
+    metaClient.getStorage().deleteDirectory(new StoragePath(basePath));
     this.tableType = tableType;
+    Properties properties = new Properties();
```

Comment:
why add these configs?

Reply:
These configs are required for the parameterised arguments to test the various cases.

Reply:
yes, the entire test was not working properly for table version 6. While investigating the failure I had to fix all these and added validation in the end that table version is as expected. prior to this patch, even though we had parametrized, table version 6 was not creating table version 6, it was all table version 8.